spec/features/comment_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "spec_helper"
+
+describe "Visitor" do
+
+  let!(:user) { create :user }
+  let!(:post) { create :post }
+
+  comment = "This is a comment"
+  reply = "This is a reply"
```

Comment:
it's standard to use the `let` syntax for creating generic variables like this as well (although I'm not entirely sure why)
```ruby
let(:comment) { "This is a comment" }
let(:reply)   { "This is a reply" }
```